components/TokenDropdownGauges.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {Fragment, ReactElement, cloneElement, useRef} from 'react';
+import {Menu, Transition} from '@headlessui/react';
+import IconChevron from 'components/icons/IconChevron';
+
+export type TDropdownGaugeOption = {
+	icon?: ReactElement;
+	label: string;
+	value: any;
+	zapVia: string;
```

Comment:
I reckon this can be optional since it will not be needed for all dropdowns
```suggestion
	zapVia?: string;
```